client/src/components/AddNewProfile.js

```diff
@@ -130,6 +130,7 @@ const AddNewProfile = () => {
       <Form.Button primary type="submit">
         Add new profile
       </Form.Button>
+      { profileCreated === false && <div>Profile has not been created</div>}
```

Comment:
```suggestion
      { profileCreated === false && <p>Something went wrong when creating the profile. Please try again.</p>}
```
1. Since this is a sentence, let's use the paragraph `p` tag since this is more semantic.
1. We can also update the message to be a bit more user friendly—we could use my suggestion